programs/temp/src/instructions/create_nft.rs

```diff
@@ -63,70 +53,82 @@ pub fn mint_nft(
     name: String,
     symbol: String,
 ) -> Result<()> {
-    let uri = "hello world";
+    let mint = &ctx.accounts.mint;
+    let mint_authority = &ctx.accounts.mint_authority;
+    let payer = &ctx.accounts.payer;
+    let system_program = &ctx.accounts.system_program;
+    let token_program = &ctx.accounts.token_program;
+    let token_metadata_program = &ctx.accounts.token_metadata_program;
+    let rent = &ctx.accounts.rent;
+    let offset_metadata = &ctx.accounts.offset_metadata;
+    let offset_tiers = &mut ctx.accounts.offset_tiers;
+    let metadata = &mut ctx.accounts.metadata;
+    let master_edition = &mut ctx.accounts.master_edition;
+
+    // TODO: check that offset_tiers.levels.len() > 0
```

Comment:
should it be equal to the global-state.levels? and if so, should that be in the anchor constraints?

Reply:
It's fine to check it here imo, removed the comment